src/Item.cpp

```diff
@@ -4,26 +4,42 @@
 namespace tt
 {
 
-    Item::Item( sf::Texture texture,
+    Item::Item( std::string id,
+                sf::Texture texture,
```

Comment:
What I mentioned above about passing by-value, applies here too. This constructor should look like:

```
Item::Item(const std::string& id, const sf::Texture& texture...
```

You get the point.

Reply:
`Vehicle(sf::Texture texture,`

Reply:
Yup, looks like I forgot one too.